Makefile

```diff
@@ -13,31 +13,69 @@
 # limitations under the License.
 
 GO := go
-
 EPOCH_TEST_COMMIT := f98810348b4b26d8c4dd64e36bde613049b2b0b9
+PROJECT := github.com/IBM/cap
+BUILD_DIR := _output
+SOURCES := $(shell find go/atom go/cap go/captn go/shared vendor/ -name '*.go')
 
 default: help
 
 help:
 	@echo "Usage: make <target>"
 	@echo
-	@echo " * 'verify'           	- Execute the source code verification tools"
-	@echo " * 'install.tools'    	- Install tools used by verify"
+	@echo " * 'binaries'      - Build binaries"
+	@echo " * 'clean'         - Clean build artifacts"
+	@echo " * 'test'          - Run Unit and Integration Tests"
+	@echo " * 'unit'          - Run Unit Tests"
+	@echo " * 'coverage'      - Determine Unit and Integration Test Coverage, generates profile in coverage.out"
+	@echo " * 'verify'        - Execute the source code verification tools"
+	@echo " * 'install.tools' - Install tools used by verify"
+
+.PHONY: binaries clean test unit integration coverage
+
+binaries: $(BUILD_DIR)/captn
```

Comment:
mark all the go output as `phony` and let the go tooling handle it.

Reply:
ok.. phony it to make it run..

Reply:
done